test/core/mfs-preload.spec.js

```diff
@@ -56,4 +72,22 @@ describe('MFS preload', () => {
       })
     })
   })
+
+  it('should disable preloading MFS', function (done) {
+    this.timeout(80 * 1000)
```

Comment:
Surely this test doesn't need a timeout adjustment?

Reply:
Have removed it.